locales/en/messages.json

```diff
@@ -6527,4 +6527,4 @@
         "message": "Reached the maximum limit of the shown presets number",
         "description": "Message that apprears on presets tab if too many presets found"
     }
-}
+}
```

Comment:
Are you sure we don't want a newline here?

Reply:
Fixed